src/backend/base/langflow/api/v1/flows.py

```diff
@@ -334,11 +336,20 @@ async def delete_multiple_flows(
 
     """
     try:
-        deleted_flows = db.exec(select(Flow).where(col(Flow.id).in_(flow_ids)).where(Flow.user_id == user.id)).all()
-        for flow in deleted_flows:
+        flows_to_delete = db.exec(select(Flow).where(col(Flow.id).in_(flow_ids)).where(Flow.user_id == user.id)).all()
```

Comment:
could you add a unit test for this endpoint ?

Reply:
sure! 💪